lib/instances/Stats_spec.lua

```diff
@@ -0,0 +1,9 @@
+local Instance = import("../Instance")
+
+describe("instances.Stats", function()
+	it("should instantiate", function()
+		local instance = Instance.new("Stats")
+
+		assert.not_nil(instance)
+	end)
```

Comment:
We should normalize these tests fairly soon to use `Foo:new()` and friends instead of `Instance.new("Foo")`, as well as removing services from `init`, renaming `init` to something else, and making `Game` instantiate the objects via `Foo:new()` as well.